README.md

```
@@ -1,12 +1,227 @@
-you may not use this file except in compliance with the License.
+you may not use this project except in compliance with the License.
 You may obtain a copy of the License at
     http://www.apache.org/licenses/LICENSE-2.0
+
+# Upstream Dependencies
+
+We gratefully acknowledge the use of the following projects which are incorporated in or used by this project:
+
+| Project | Link | License |
+|---------|------|---------|
+| Docker | https://www.docker.com | Apache-2.0 |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)